packages/fxa-auth-server/config/dev.json

```diff
@@ -426,7 +426,7 @@
     "secondInterval": "2s"
   },
   "mjml": {
-    "enabledEmailAddress": "@mozilla.com$",
-    "templates":  []
+    "enabledEmailAddress": "@testuser.com$",
```

Comment:
I figured that locally we're using this address to send emails so it's just pushed to test the changes